src/Acuminator/Acuminator.Tests/packages.config

```diff
@@ -22,18 +22,18 @@
   <package id="System.Globalization" version="4.3.0" targetFramework="net461" />
   <package id="System.IO.FileSystem" version="4.0.1" targetFramework="net461" />
   <package id="System.IO.FileSystem.Primitives" version="4.0.1" targetFramework="net461" />
-  <package id="System.Linq" version="4.3.0" targetFramework="net461" />
-  <package id="System.Linq.Expressions" version="4.3.0" targetFramework="net461" />
-  <package id="System.Reflection" version="4.3.0" targetFramework="net461" />
+  <package id="System.Linq" version="4.3.0" targetFramework="net461" requireReinstallation="true" />
+  <package id="System.Linq.Expressions" version="4.3.0" targetFramework="net461" requireReinstallation="true" />
+  <package id="System.Reflection" version="4.3.0" targetFramework="net461" requireReinstallation="true" />
   <package id="System.Reflection.Metadata" version="1.2.0" targetFramework="net461" />
   <package id="System.Reflection.Primitives" version="4.3.0" targetFramework="net461" />
   <package id="System.Resources.ResourceManager" version="4.3.0" targetFramework="net461" />
-  <package id="System.Runtime" version="4.3.0" targetFramework="net461" />
-  <package id="System.Runtime.Extensions" version="4.3.0" targetFramework="net461" />
+  <package id="System.Runtime" version="4.3.0" targetFramework="net461" requireReinstallation="true" />
```

Comment:
Why do we need `requireReinstallation="true"` here for all these packages?

Reply:
Frankly speaking, I have no idea because this config file were auto generated when I tried to emove System.ValueTuples package